h2o-py/h2o/grid/grid_search.py

```diff
@@ -694,11 +706,33 @@ def gini(self, train=False, valid=False, xval=False):
         :param bool valid: If valid is True, then return the Gini Coefficient value for the validation data.
         :param bool xval:  If xval is True, then return the Gini Coefficient value for the cross validation data.
 
-        :returns: The Gini Coefficient for this binomial model.
+        :returns: The Gini Coefficient for the models in this grid.
         """
         return {model.model_id: model.gini(train, valid, xval) for model in self.models}
 
 
+    # @alias('pr_auc')
+    def aucpr(self, train=False, valid=False, xval=False):
+        """
+        Get the aucPR (Area Under PRECISION RECALL Curve).
+
+        If all are False (default), then return the training metric value.
+        If more than one options is set to True, then return a dictionary of metrics where the keys are "train",
+        "valid", and "xval".
+
+        :param bool train: If train is True, then return the aucpr value for the training data.
+        :param bool valid: If valid is True, then return the aucpr value for the validation data.
+        :param bool xval:  If xval is True, then return the aucpr value for the validation data.
+
+        :returns: The AUCPR for the models in this grid.
+        """
+        return {model.model_id: model.aucpr(train, valid, xval) for model in self.models}
+
+    @deprecated(replaced_by=aucpr)
+    def pr_auc(self):
+        pass
+
```

Comment:
seb:  The giniCoef() is supposed to be deprecated as well.  The new one is gini().  However, it was not done properly and is not working.  Would it be too painful for you to fix it as well?

Reply:
Seb:

I gave up.  I checked R, the current version is h2o.giniCoef() while python wants gini().  Please ignore what I said before.  I will write a new JIRA for the gini.

Reply:
Here is the JIRA: https://0xdata.atlassian.net/browse/PUBDEV-6888